src/composition/models/stacking.jl

```diff
@@ -54,7 +58,7 @@ const Stack{modelnames, inp_scitype, tg_scitype} =
             ProbabilisticStack{modelnames, inp_scitype, tg_scitype}}
 
 """
-    Stack(;metalearner=nothing, resampling=CV(), name1=model1, name2=model2, ...)
+    Stack(;metalearner=nothing, resampling=CV(), name1=model1, cache=true, acceleration=CPU1(), name2=model2, ...)
```

Comment:
This doc-string got a little mangled, with the model1 and model2 being separated. As this has grown passed the length of the line, how about we not list all the options here:

```suggestion
    Stack(;metalearner=nothing, name1=model1, name2=model2, keyword_options...)
```